ItemChanger/Events/LifecycleEvents.cs

```diff
@@ -63,22 +74,37 @@ public class Invoker
     {
         internal Invoker() { }
 
-        internal void NotifyHooked() => OnItemChangerHook?.Invoke();
+        internal static void InvokeList(List<Action> list, [CallerMemberName] string caller = "")
```

Comment:
This should probably either be private or in another class so it's reusable. Speaking of, is there a reasonable way to reuse this for game events as well?